README.md

```diff
@@ -359,7 +359,7 @@ An account owner may create a long-lived API Token using the following AWS CLI c
 ```bash
 AWS_REGION="us-east-1"  # change to your deployment region
 token_string="YOUR_STRING_HERE"  # change to a unique string for a user
-aws --region $AWS_REGION dynamodb put-item --table-name LISAApiTokenTable \
+aws --region $AWS_REGION dynamodb put-item --table-name <deployment name>-LISAApiTokenTable \
```

Comment:
To nitpick on these, can you make the deployment be a variable so it's copy/paste-able (and same with the rest of the commands). it's ok if the variable is hardcoded to be `deployment=dev`

Reply:
Can do!

Reply:
Ah actually the envvar `DEPLOYMENT_NAME` is defined as a part of the setup previously in the README. I'll update these to use that.